native/cocos/audio/android/UrlAudioPlayer.cpp

```diff
@@ -242,7 +242,7 @@ bool UrlAudioPlayer::prepare(const std::string &url, SLuint32 locatorType, std::
                              int length) {
     _url = url;
     _assetFd = assetFd;
-
+#if CC_PLATFORM == CC_PLATFORM_ANDROID
```

Comment:
Why can't we set volume on openharmony?

Reply:
Openharmony does not support URL playback

Reply:
then just change the behavior in AudioPlayerProvider, return all players as pcmPlayer. cause less problem

Reply:
Yes but this will cause compilation issues, so don't modify it for now